rr/gstreamer/gstmedia.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from gi.repository import Gst as gst
+from gi.repository import GLib
+import gi
+gi.require_version('Gst', '1.0')
```

Comment:
Fix warning that appears at runtime.

Reply:
If you need the indenter to ignore a line, place `# nopep8` at the end of the line.